srcpkgs/meli/patches/fixes-for-32bit-systems.patch

```diff
@@ -0,0 +1,41 @@
+--- melib/src/backends/notmuch.rs
```

Comment:
Have these been upstreamed?

Reply:
Not yet. It seems that the author doesn't know about the existence of 32-bit archs.